examples/multi-callback-all/index.ts

```diff
@@ -0,0 +1,165 @@
+import * as azure from "@pulumi/azure";
+
+const resourceGroup = new azure.core.ResourceGroup("example", { location: azure.Locations.WestUS2 });
+
+// HTTP
+const httpFunc = new azure.appservice.HttpFunction("http", {
+    callback: async (context, request) => ({ status: 200, body: "Hello World" }),
+});
+
+// Fire on schedule
+const timerFunc = new azure.appservice.TimerFunction("timer", {
+    schedule: { minute: 0 },
+    callback: async (context, timer) => {
+        console.log(`Running at ${timer}`);
+    },
+});
+
+// Storage Queue
+const storageAccount = new azure.storage.Account("storage", {
+    resourceGroupName: resourceGroup.name,
+    accountReplicationType: "LRS",
+    accountTier: "Standard",
+});
+
+const queue = new azure.storage.Queue("queue", {
+    resourceGroupName: resourceGroup.name,
+    storageAccountName: storageAccount.name,
+});
+
+const storageQueueFunc = new azure.storage.QueueFunction("storage-queue", {
```

Comment:
question for later, but what do we need this name for?

Reply:
Each Azure Function has a name (visible in the portal, logs, management API, etc)

Reply:
yup.  LGTM .